SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md

```diff
@@ -9,8 +9,8 @@ Before porting your project:
 * If you haven't done this already, install SpatialOS and the GDK's dependencies and clone the SpatialOS Unreal Engine fork by following:
     * [Getting started: 1 - Dependencies]({{urlRoot}}/content/get-started/dependencies)
     * [Getting started: 2 - Get and build the GDK’s Unreal Engine Fork]({{urlRoot}}/content/get-started/build-unreal-fork)
-* If you haven't done this already, clone the GDK repository by following:
-    *  [Set up the GDK and Starter Project]({{urlRoot}}/content/get-started/gdk-and-starter-project) (You only need to clone the GDK repository, not the Starter Project repository.)  
+* If you haven't done this already, clone the GDK repository into your `Plugins` folder by following the steps for cloning the GDK in:
+    *  [Set up the GDK and Starter Project]({{urlRoot}}/content/get-started/gdk-and-starter-project) (You only need to clone the GDK repository, not the `StarterProject` repository.)  
```

Comment:
(You only need to clone the GDK repository, not the `StarterProject` repository.)  Is completely incorrect.